app/controllers/application_controller.rb

```diff
@@ -47,4 +60,6 @@ def api_url_method
       method(:content_item_url)
     end
   end
+
+  class InvalidRequest < RuntimeError; end
```

Comment:
Is this needed here?  It seems to be a duplicate of lines 3 and 11.

Reply:
Oops, thanks for spotting that. Just something I accidentally left over.

Reply:
It looks like line 11 can be removed too (as the class is already declared on line 3)

Reply:
Argh! sorry. I seem to be an idiot today.

This may have looked like a test of your attention to detail on a PR but it is just me making a mess. Thanks again for catching.